app.py

```diff
@@ -22,31 +14,21 @@ def show_todo_form():
 
 @app.route("/todo/set/", methods=['POST'])
 def recover_data():
-    global todoList
+    todoArray = []
     todoTask = request.form['todoInput']
-    todoHeader = request.form['todoHeader'] 
-
+    todoHeader = request.form['todoHeader']
     if todoHeader in todoDict.keys():
-        todoList = todoDict[todoHeader] 
-        todoList.append(todoTask)
-    else: 
-        todoList.append(todoTask)
-        todoDict[todoHeader] = todoList
-    
-    todoList = []
-    return render_template("todo.html", data=todoDict) 
+        todoDict[todoHeader].append(todoTask)
+    else:
+        todoArray.append(todoTask)
+        todoDict[todoHeader] = todoArray
```

Comment:
We could simplify not using `todoArray` with:
```
todoDict[todoHeader] = [todoTask]
```

Reply:
Alright, already fixed !